src/main/resources/org/jenkinsci/plugins/workflow/steps/UnstableStep/config.jelly

```diff
@@ -28,4 +28,7 @@ THE SOFTWARE.
     <f:entry field="message" title="${%Message}">
         <f:textbox/>
     </f:entry>
+    <f:entry field="markBuildResult" title="${%Don't set build result}">
```

Comment:
Just avoid the contraction:

```suggestion
    <f:entry field="markBuildResult" title="${%Do not set build result}">
```

`'` can be interpreted as a metacharacter by `MessageFormat`. Cannot recall if that is a risk in this case, but best avoided. If you must use an apostrophe,
```suggestion
    <f:entry field="markBuildResult" title="${%Don’t set build result}">
```